src/stactools/sentinel2/stac.py

```diff
@@ -145,6 +146,12 @@ def create_item(granule_href: str,
             f'Error populating MGRS and Grid Extensions fields from ID: {metadata.scene_id}'
         )
 
+    # View Extension
+    view = ViewExtension.ext(item, add_if_missing=True)
+    view.sun_azimuth = metadata.metadata_dict.get("s2:mean_solar_azimuth")
+    if msz := metadata.metadata_dict.get("s2:mean_solar_zenith"):
```

Comment:
I think the previous `s2` fields can be removed with the inclusion of the equivalent fields in the view ext

Reply:
done.